src/collective/easyform/interfaces/fields.py

```diff
@@ -85,6 +85,17 @@ class IFieldExtender(Schema):
         required=False,
         default=False,
     )
+    directives.write_permission(depends_on=config.EDIT_TALES_PERMISSION)
```

Comment:
Isn't  the permission too restrictive. IIRC only Manager would be able to modify/set this.

Reply:
it's the same permission as for the others fields ;)

Reply:
The pattern comes from patternslib, in Plone 5 you need the extra requirement of plone.patternslib and in Plone 6 this will be available by default.